index.html

```diff
@@ -1238,27 +1238,25 @@ <h1>Core Properties</h1>
 including whether these properties are required or optional.
     </p>
 
-    <section>
-      <h2>
-Identifiers
-      </h2>
-      <p>
+    <p class="note">
 Identifiers are used in the data model to identify specific instances of people,
 organizations, devices, keys, services, and things in general. Identifiers are
-typically URLs, or more generally, <a>URIs</a>. Non-<a>URI</a>-based identifiers
-are not advised because, while the data model supports them, they are not easy
-to use with other Internet-based identifiers.
-      </p>
-    </section>
+typically URLs, or more generally, <a>URIs</a> (and in some cases specifically
```

Comment:
Since this section is being cleaned up a bit, it seems we should say that "identifiers SHOULD be URIs" instead of this other non-enforceable language around "identifiers are typically URLs". And then we can give the advice that follows. Also, we may want to say that if an identifier is *not* a URI, then globally unambiguous interop is not possible.

Reply:
agree... identifiers should be URIs.